collect_app/src/main/java/org/odk/collect/android/provider/FormsProvider.java

```diff
@@ -528,14 +528,12 @@ public int update(Uri uri, ContentValues values, String where,
         return newWhereArgs;
     }
 
-    // Leading slashes are removed from paths to support minSdkVersion < 18:
-    // https://developer.android.com/reference/android/content/UriMatcher
     static {
         URI_MATCHER = new UriMatcher(UriMatcher.NO_MATCH);
-        URI_MATCHER.addURI(FormsProviderAPI.AUTHORITY, UriUtils.stripLeadingUriSlashes(FormsColumns.CONTENT_URI.getPath()), FORMS);
-        URI_MATCHER.addURI(FormsProviderAPI.AUTHORITY, UriUtils.stripLeadingUriSlashes(FormsColumns.CONTENT_URI.getPath()) + "/#", FORM_ID);
+        URI_MATCHER.addURI(FormsProviderAPI.AUTHORITY, FormsColumns.CONTENT_URI.getPath(), FORMS);
```

Comment:
I think it's a wrong solution please take a look at this commit where it was implemented https://github.com/opendatakit/collect/commit/f22b49d0e600824447aefd6dfa69dbf96ecddebc and InstancesProvider class.
i think it should be:
`forms`
`forms/#`
`uniqueFormsByFormId`

Reply:
Hie @grzesiek2010! Thanks for directing to the commit. I reverted it to how it was implemented.

Reply:
I made a mistake and what you did here was right. Could you add it back?
Sorry my mistake... Please just revert this https://github.com/opendatakit/collect/pull/3625/commits/005536279258cd4dd0b51772925a9e825bf9e696

Reply:
Yes that's okay @grzesiek2010! I'll revert it.